src/seeders/20201211172017-grants.js

```diff
@@ -85,12 +87,28 @@ const grants = [
   },
 ];
 
+const baseGrant = {
+  startYear: 2021,
+  startDate: '2021-01-01',
+  status: 'Active',
+  endDate: '2032-09-01',
+};
+
+const programs = grants.map((grant) => ({
+  ...baseGrant,
+  name: faker.company.companyName(),
+  id: grant.id,
+  grantId: grant.id,
+}));
```

Comment:
Should we also add the program types since that will be more aligned with what is in prod?